pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -1351,6 +1352,11 @@ async def parse_advisories(self, result):
 
                 for collection in update.collections:
                     coll_dict = UpdateCollection.createrepo_to_dict(collection)
+                    if coll_dict["name"] is None:
+                        coll_dict["name"] = (
+                            "collection-"
+                            + hashlib.sha256(update_record.fromstr.encode("utf-8")).hexdigest()
```

Comment:
There is also the possibility to use maybe `.md5()` to produce a shorter hash.

Reply:
It's possible to use but if you want a shorter hash I would recommend just taking a prefix from a sha256.